CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
```

Comment:
This isn't your fault but there are two `### Changed` sections in the CHANGELOG.

Reply:
I'll check on that